.github/workflows/wheels.yml

```diff
@@ -111,11 +111,12 @@ jobs:
           - cu121
           - cu124
           - rocm6.1
```

Comment:
@johnnynunez @bottler it looks like only rocm6.3 wheel is being built by default but all rocm6.x wheels are tried for upload

Reply:
with rocm may not like ' ', and should be “”? I have no idea what is happening, seems that in rocm xformer is building for 6.2 also

Reply:
https://github.com/facebookresearch/xformers/blob/main/.github/workflows/wheels.yml#L45

Reply:
> https://github.com/facebookresearch/xformers/blob/main/.github/workflows/wheels.yml#L45

good catch